P/PETSc/build_tarballs.jl

```diff
@@ -0,0 +1,65 @@
+using BinaryBuilder
+
+name = "PETSc"
+version = v"3.13.0"
+
+# Collection of sources required to build PETSc. Avoid using the git repository, it will
+# require building SOWING which fails in all non-linux platforms.
+sources = [
+    ArchiveSource("http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-3.13.0.tar.gz",
+    "df2ff7cb0341bb534a18c7dbea37aa2e2c543a440bf63c24977a605d9b5f8324"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/petsc*
+libinclude="${prefix}/include"
+atomic_patch -p1 $WORKSPACE/srcdir/patches/petsc_name_mangle.patch
+
+if [[ $nbits == 64 ]] && [[ "$target" != aarch64-* ]]; then
+  BLAS_LAPACK_LIB="${libdir}/libopenblas64_.${dlext}"
+  BLAS_LAPACK_SUFFIX="_64_"
+  blas_64=1
+else
+  BLAS_LAPACK_LIB="${libdir}/libopenblas.${dlext}"
+  BLAS_LAPACK_SUFFIX=""
+  blas_64=0
+fi
+
+if [[ ${target} != *darwin* ]]; then
+    # Needed to find libgfortran for OpenBLAS.
+    export CFLAGS="-Wl,-rpath-link,/opt/${target}/${target}/lib -Wl,-rpath-link,/opt/${target}/${target}/lib64"
+fi
+
+opt_flags="--with-debugging=0 COPTFLAGS='-O3 -march=native -mtune=native' -CXXOPTFLAGS='-O3 -march=native -mtune=native' FOPTFLAGS='-O3 -march=native -mtune=native'"
```

Comment:
I'm not sure we should use all these `native` options, otherwise we optimise exclusively for the the microarchitecture of the machine where the build runs

Reply:
OK, I added them based on the documentation from PTSEc. Certainly, some can be too specific

Reply:
Yes both `native` options should be removed.